src/Components/CreateImageWizard/formComponents/Repositories.js

```diff
@@ -99,7 +99,7 @@ const BulkSelect = ({
   );
 };
 
-// Utility function to convert from Content Sources to Image Builder API schema
+// Utility function to convert from Content Sources to Image Builder payload repo API schema
 const convertSchemaToImageBuilder = (repo) => {
```

Comment:
Can we change this to convertSchemaToIBPayloadRepo?

Reply:
absolutely